dnf.spec

```diff
@@ -111,11 +111,8 @@ BuildRequires:  python-iniparse
 BuildRequires:  python-libcomps >= %{libcomps_version}
 BuildRequires:  python-librepo >= %{librepo_version}
 BuildRequires:  python-nose
-%if 0%{?rhel} && 0%{?rhel} <= 7
```

Comment:
I wrote above.. pygpgme is dead completely, I had to maintain some number of patches to keep it compatible with modern libraries and now it's failing to build again and I'm not going to fix. upstream is dead. I've requested maintainers in RHEL to update gpgme which will contain python bindings.